From: ChepKun <ProgrammerKun@gmail.com>
Subject: [PATCH] hardware/qcom/media-legacy (0001): QCMetadata: add keys for FLAC meta data

diff --git a/mm-core/inc/QCMetaData.h b/mm-core/inc/QCMetaData.h
index 3ede20d..c54c357 100644
--- a/mm-core/inc/QCMetaData.h
+++ b/mm-core/inc/QCMetaData.h
@@ -65,6 +65,13 @@ enum {
     kKeyUseArbitraryMode     = 'ArbM',  //bool (int32_t)
     kKeySmoothStreaming      = 'ESmS',  //bool (int32_t)
     kKeyHFR                  = 'hfr ',  // int32_t
+
+    kKeySampleBits        = 'sbit', // int32_t (audio sample bit-width)
+    kKeyMinBlkSize        = 'mibs', //int32_t
+    kKeyMaxBlkSize        = 'mabs', //int32_t
+    kKeyMinFrmSize        = 'mifs', //int32_t
+    kKeyMaxFrmSize        = 'mafs', //int32_t
+    kKeyMd5Sum            = 'md5s', //cstring
 };
 
 #if 0
